README.rst

```diff
@@ -1 +1,29 @@
-# waiting for andrew to pr a good readme
\ No newline at end of file
+telethon-tgcrypto
+-----
+
+Module ``telethon-tgcrypto`` brings better performance to
+`Telethon <https://github.com/LonamiWebs/Telethon>`__ lib.
+
+Installation
+------------
```

Comment:
This and credits should probably be smaller headers, but I'm not sure how RST works exactly